ocs_ci/ocs/must_gather/const_must_gather.py

```diff
@@ -0,0 +1,554 @@
+    'system:openshift:operator:kube-controller-manager-recovery.yaml',
+    'system:openshift:operator:kube-scheduler-recovery.yaml',
+    'system:openshift:operator:kube-scheduler:public-2.yaml',
+    'system:openshift:operator:kube-storage-version-migrator-operator.yaml',
+    'system:openshift:operator:openshift-apiserver-operator.yaml',
+    'system:openshift:operator:openshift-config-operator.yaml',
+    'system:openshift:operator:openshift-controller-manager-operator.yaml',
+    'system:openshift:operator:openshift-etcd-installer.yaml',
+    'system:openshift:operator:openshift-kube-apiserver-installer.yaml',
+    'system:openshift:operator:openshift-kube-controller-manager-installer.yaml',
+    'system:openshift:operator:openshift-kube-scheduler-installer.yaml',
+    'system:openshift:operator:service-ca-operator.yaml',
+    'system:openshift:public-info-viewer.yaml',
+    'system:openshift:public-info-viewer.yaml',
+    'system:openshift:scc:anyuid.yaml',
+    'system:openshift:scc:hostaccess.yaml',
+    'system:openshift:scc:hostmount.yaml',
+    'system:openshift:scc:hostnetwork.yaml',
+    'system:openshift:scc:nonroot.yaml',
+    'system:openshift:scc:privileged.yaml',
+    'system:openshift:scc:restricted.yaml',
+    'system:openshift:templateservicebroker-client.yaml',
+    'system:openshift:tokenreview-openshift-controller-manager.yaml',
+    'system:openshift:tokenreview-openshift-controller-manager.yaml',
+    'system:persistent-volume-provisioner.yaml',
+    'system:public-info-viewer.yaml',
+    'system:public-info-viewer.yaml',
+    'system:registry.yaml',
+    'system:router.yaml',
+    'system:scope-impersonation.yaml',
+    'system:scope-impersonation.yaml',
+    'system:sdn-manager.yaml',
+    'system:sdn-reader.yaml',
+    'system:sdn-readers.yaml',
+    'system:volume-scheduler.yaml',
+    'system:volume-scheduler.yaml',
+    'system:webhook.yaml',
+    'system:webhooks.yaml',
+    'telemeter-client-view.yaml',
+    'telemeter-client.yaml',
+    'telemeter-client.yaml',
+    'thanos-querier.yaml',
+    'thanos-querier.yaml',
+    'version.yaml',
+    'view.yaml',
+    'volumesnapshotclass',
+    'whereabouts-cni.yaml',
+]
+
+GATHER_COMMANDS_CEPH_4_7 = []
+
+GATHER_COMMANDS_JSON_4_7 = []
+
+GATHER_COMMANDS_OTHERS_4_7 = []
+
+GATHER_COMMANDS_VERSION = {
+    ('CEPH', 4.5): GATHER_COMMANDS_CEPH + GATHER_COMMANDS_CEPH_4_5,
+    ('JSON', 4.5): GATHER_COMMANDS_JSON + GATHER_COMMANDS_JSON_4_5,
+    ('OTHERS', 4.5): GATHER_COMMANDS_OTHERS + GATHER_COMMANDS_OTHERS_4_5,
+    ('CEPH', 4.6): GATHER_COMMANDS_CEPH + GATHER_COMMANDS_CEPH_4_6,
+    ('JSON', 4.6): GATHER_COMMANDS_JSON + GATHER_COMMANDS_JSON_4_6,
+    ('OTHERS', 4.6): GATHER_COMMANDS_OTHERS + GATHER_COMMANDS_OTHERS_4_6,
+    ('CEPH', 4.7): GATHER_COMMANDS_CEPH + GATHER_COMMANDS_CEPH_4_7,
+    ('JSON', 4.7): GATHER_COMMANDS_JSON + GATHER_COMMANDS_JSON_4_7,
+    ('OTHERS', 4.7): GATHER_COMMANDS_OTHERS + GATHER_COMMANDS_OTHERS_4_7,
```

Comment:
I see that 4.6 has big list of GATHER_COMMANDS_OTHERS_4_6 and you combine it with GATHER_COMMANDS_OTHERS.  But here for 4.7 you have empty lists. Shouldn't you now expect at least the same as we have in 4.6 like I suggested above to use 4.6 lists for 4.7 as of now?

And once we will know the changes we will do required changes?

Or some other approach?

Reply:
Done 
```
GATHER_COMMANDS_CEPH_4_7 = GATHER_COMMANDS_CEPH_4_6

GATHER_COMMANDS_JSON_4_7 = GATHER_COMMANDS_JSON_4_6

GATHER_COMMANDS_OTHERS_4_7 = GATHER_COMMANDS_OTHERS_4_6
```